.github/workflows/tag_release.yaml

```diff
@@ -0,0 +1,32 @@
+name: tag-release
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "config/version.txt"
+
+jobs:
+  tag:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3.0.2
+        with:
+          fetch-depth: 0
+      - name: Get current version from file
+        id: tag
+        run: |
+          export VERSION="$(cat config/version.txt)"
+          echo "::set-output name=tag::$VERSION"
```

Comment:
Should be changed because of https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/